zarr_etl_tools/utils/metadata.py

```diff
@@ -728,3 +830,13 @@ def remove_unwanted_fields(self, dataset: xr.Dataset) -> xr.Dataset:
         dataset[self.data_var()].encoding["compressor"] = compressor
 
         return dataset
+
+
+class StacType(Enum):
+    ITEM = "datasets"
+    COLLECTION = "collections"
+    CATALOG = ""
+
+
+def get_local_stac_path(title: str, stac_type: StacType) -> pathlib.Path:
```

Comment:
WiIl consider. I tried this at first and it mostly introduced complexity, since how you use the function would vary depending on the backing store. Let me see if I can find a way.